From 942c27a4a5fd53edde9c2ef2a6ae41009b5a62c6 Mon Sep 17 00:00:00 2001
From: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
Date: Thu, 31 Oct 2024 14:59:09 -0300
Subject: [PATCH] build/config/compiler/BUILD.gn: Disable external-clangrt

Disable external-clangrt feature as it's not support by the current
clang toolchain version.

Signed-off-by: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
---
 build/config/compiler/BUILD.gn | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index c5949310d7b3a..38b1211e65b99 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1023,9 +1023,6 @@ config("compiler") {
     # For deterministic builds, keep the local machine's current working
     # directory from appearing in build outputs.
     "-Zremap-cwd-prefix=.",
-
-    # We use clang-rt sanitizer runtimes.
-    "-Zexternal-clangrt",
   ]
 
   if (!is_win || force_rustc_color_output) {
-- 
2.45.2

